alignak/objects/satellitelink.py

```diff
@@ -91,9 +89,10 @@ class SatelliteLink(Item):
         'attempt':              StringProp(default=0, fill_brok=['full_status']),
 
         # can be network ask or not (dead or check in timeout or error)
-        'reachable':            BoolProp(default=False, fill_brok=['full_status']),
-        'last_check':           IntegerProp(default=0, fill_brok=['full_status']),
-        'managed_confs':        StringProp(default={}),
+        'reachable': BoolProp(default=True, fill_brok=['full_status']),
```

Comment:
Because in dispatcher, I modified the behavior.

By default we prepare conf to a satellite (scheduler, poller...)

After we send, it is reachable by default, so we begin attempts, if not ok at xx attempts, we prepare for another satellite (if we have many) or we need wait satellite be up